gradlew

```diff
@@ -1,78 +1,129 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
This file shouldn't be `ISO-8859` encoded. When I ran `../gradle-7.3.1/bin/gradle wrapper` locally, it regenerated it as UTF-8.

Reply:
That's strange 🤔 I was doing `./gradlew wrapper --gradle-version 7.3.2 --distribution-type all`